### DISCUSSION TITLE
有计划支持更改 WebDAV 的CORS 设置吗

### DISCUSSION THREAD

**Discussion by heddxh** [AlistGo/alist#986] [General]
如题。部分程序受制于cors 导致无法连接。

**Comment by anwen-anyi** [comment]
cors 我记得是网盘程序提供的吧,而不是Alist程序提供的..

**Reply by heddxh** [reply]
我是指 alist 自己的WebDAV

**Reply by anwen-anyi** [reply]
Alist的webdav不得调用网盘的吗? （我也不太懂喔~）

**Reply by heddxh** [reply]
不啊。比如我用本地策略，那肯定是alist提供的

**Reply by anwen-anyi** [reply]
> 不啊。比如我用本地策略，那肯定是alist提供的

emm 那就不太了解了~我主要调用一些常用的小软件或者各个网盘的音乐.. 对于这些没太深度使用哈哈..

**Comment by xhofe** [comment]
请详细描述。

**Reply by heddxh** [reply]
Obsidian 的 Android 端本质是一个浏览器，在其内部用 Webdav 同步数据收到 CORS 限制。因此我需要能在 Alist 的Webdav 设置中更改跨域配置，类似于
```
AllowOrigins = *
AllowMethods = *
AllowHeaders = *
AllowCredentials = true
```
（我也不太懂，不知道有没有讲清楚）

**Reply by Tom-Plants** [reply]
在基于web的app上面去引入alist webdav网盘就会碰到这个问题。目前可以的解决方案就是nginx去添加头。但不太清楚alist webdav作为一个内容分发的平台不引入跨域头的原因。
